parameter_files/fates_params_default.cdl

```diff
@@ -1079,7 +1078,7 @@ data:
 
  fates_cnp_eca_km_ptase = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 ;
 
- fates_cnp_eca_lambda_ptase = 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 ;
+ fates_cnp_eca_lambda_ptase = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ;
```

Comment:
@rgknox Similar to my comment above, would you mind confirming this change?

Reply:
for this, the value is not used at all, and there is no equivalent used in the methods. Maybe it would be better to remove the parameter completely, but I figured I would leave it in case we change that in the near future.  Perhaps instead of 0's for these, we should use a value that implies a do not use.. like -999?

Reply:
If the parameter is not used by default, then I think it is fine as is. Perhaps in the future we could have a separate PR that sets all unused parameters to NaN.